docs/blockchain_schema/version1.ts

```diff
@@ -0,0 +1,61 @@
+// Version 1 schema
+
+type Multihash = string;
+type HashPointer = string;
+type UID = string;
+type Datetime = string;
+
+// Block schema
+interface Block {
+  sequence: number; // 0-indexed
+  // The datetime at which the earliest version of this block was created.
+  origin_created: Datetime; // new
```

Comment:
What is this expressed as? Some kinda ISO date string? Comment on Datetime above would be nice